src/geophires_x/Reservoir.py

```diff
@@ -74,10 +74,9 @@ def __init__(self, model: Model):
 
         self.Tmax = self.ParameterDict[self.Tmax.Name] = floatParameter(
             "Maximum Temperature",
-            value=400.0,
             DefaultValue=400.0,
             Min=50,
-            Max=400,
+            Max=500,
```

Comment:
Max increased to 500 - note default remains at 400 so existing inputs are not affected